app/src/main/java/com/sunasterisk/fooddaily/data/model/FoodDetail.kt

```diff
@@ -17,14 +17,14 @@ private const val JSON_KEY_STEP = "step"
 private const val JSON_KEY_STEPS = "steps"
 
 data class FoodDetail(
-    val id: Int,
-    val title: String,
-    val price: String,
-    val readyMinutes: Int,
-    val summary: String,
-    val imageUrl: String,
-    val ingredients: List<Ingredient>,
-    val instructions: List<String>
+    var id: Int,
+    var title: String,
+    var price: String?,
+    var readyMinutes: Int?,
+    var summary: String?,
+    var imageUrl: String,
+    var ingredients: List<Ingredient>?,
+    var instructions: List<String>?
```

Comment:
thêm giá trị default vào  
```suggestion
    var price: String? = null,
    var readyMinutes: Int? = null,
    var summary: String?,
    var imageUrl: String,
    var ingredients: List<Ingredient>?,
    var instructions: List<String>?
```